src/components/IIIFThumbnail.js

```diff
@@ -162,14 +161,12 @@ export class IIIFThumbnail extends Component {
             </Typography>
           </div>
         )}
-        {children}
```

Comment:
Removing the `children` prop rendering here might also be considered a breaking change. Can we please add this back?